modules/@apostrophecms/admin-bar/index.js

```diff
@@ -196,17 +188,21 @@ module.exports = {
       // user only sees one of them, etc. Called by `afterInit`
 
       groupItems() {
-        // Implement the groups and addGroups options. Mark the grouped items with a
-        // `menuLeader` property.
-        _.each(self.options.groups || self.groups.concat(self.options.addGroups || []), function (group) {
-          if (group.label) {
-            self.groupLabels[group.items[0]] = group.label;
-            group = group.items;
+        // Implement the groups and addGroups options. Mark the grouped items
+        // with a `menuLeader` property.
+        const groups = self.options.groups || self.groups.concat(self.options.addGroups || []);
+
+        groups.forEach(function (group) {
```

Comment:
This ended up not being critical to my grouping problems, but setting `group = group.items` was confusing and seemed like odd obfuscation.